app/models/role.rb

```diff
@@ -0,0 +1,7 @@
+class Role < ApplicationRecord
+  enum name: {
+    super_admin: "super_admin",
+    admin: "admin",
+    analyst: "analyst"
```

Comment:
Let's think with Claudio about the wordages we use. Admin/analyst? Does that still make sense on a per-scope basis? What about view/manage? read/write? (ok maybe not read/write...)